langserver/handler.go

```diff
@@ -37,7 +37,7 @@ type LangHandler struct {
 	symbolCache    cache
 
 	// cache the reverse import graph
-	importGraphOnce sync.Once
+	importGraphOnce *sync.Once
```

Comment:
Please add a comment to explain why this is a pointer and not a value.

It would also be nice to add a comment to mu to describe what it does and does not protect.